api/v1alpha1/workflow_types.go

```diff
@@ -120,7 +120,12 @@ type Template struct {
 	Name     string       `json:"name"`
 	Type     TemplateType `json:"templateType"`
 	Duration *string      `json:"duration,omitempty"`
-	Tasks    []string     `json:"tasks,omitempty"`
+	// +optional
+	Task *Task `json:"task,omitempty"`
```

Comment:
Definitely YES. Any suggestion?

Reply:
TaskList?

Reply:
not good enough, `TaskList` is similar to `Tasks`.

How about using `Children []string` instead of `Tasks []string`? How do you think? @YangKeao @g1eny0ung